src/App.tsx

```diff
@@ -1,29 +1,58 @@
-import React from 'react';
+import { useState } from 'react';
+
 import './App.css';
 import { getNumbers } from './utils';
 
+import { Pagination } from './components/Pagination';
+import { ListOfItems } from './components/ListOfItems/ListOfItems';
+
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
Great job!) One correction - You are importing the v4 function from the 'uuid' library, but you have not installed the library or used it.